test/index.js

```diff
@@ -67,33 +64,34 @@ describe('mkdirpStream', function () {
     });
   });
 
-  it('exports a main function, .obj and .withMode methods', function (done) {
+  it('exports a main function', function (done) {
     expect(typeof mkdirpStream).toEqual('function');
-    expect(typeof mkdirpStream.obj).toEqual('function');
     done();
   });
 
   it('takes a string to create', function (done) {
-    function assert() {
+    function assert(err) {
       expect(statMode(outputDirpath)).toBeDefined();
+      done(err);
     }
 
-    pipe([from(['test']), mkdirpStream(outputDirpath), concat(assert)], done);
+    pipeline(Readable.from(['test']), mkdirpStream(outputDirpath), assert);
   });
 
   it('takes a resolver function that receives chunk', function (done) {
-    var expected = Buffer.from('test');
+    var expected = 'test';
 
     function resolver(chunk, cb) {
-      expect(chunk.equals(expected)).toEqual(true);
+      expect(chunk).toEqual(expected);
```

Comment:
Did jest change something? This was broken before.

Or maybe it is because the new `Readable.from` doesn't turn strings into buffers. Is it fine to use strings? I technically don't consider a string to be a chunk (in node streams terms).

Reply:
I think this is because `streamx.Readable.from` does not convert strings to buffers where `miss.from` does.  I'm unsure what `require('stream').Readable.from` does.

Reply:
Right. I guess the benefit of streamx is that we don't need to care about the data type passing through the stream, so I don't think we need to wrap the test data in a buffer.

The one caveat to the above is that we need to account for buffers when testing highWaterMark stuff since they are handled a bit differently than every other data type. This library doesn't test hwm stuff though.